pkg/util/kubernetes/apiserver/apiserver_nocompile.go

```diff
@@ -20,7 +20,19 @@ var (
 )
 
 // GetPodServiceNames is used when the API endpoint of the DCA to get the services of a pod is hit.
-func GetPodServiceNames(nodeName string, podName string) []string {
+func GetPodServiceNames(nodeName string, podName string) ([]string, error) {
```

Comment:
I'm wondering if it's possible to register the endpoints behind a build flag so we don't have to do this (and handle all unmatched route with 404, which should be default)

Reply:
I added a card in our backlog.